pkg/client/ocm/client.go

```diff
@@ -509,3 +510,15 @@ func (c *client) CreateMachinePool(clusterID string, machinePool *clustersmgmtv1
 
 	return createdMachinePool, nil
 }
+
+func (c client) GetQuotaCost(organizationID string, fetchRelatedResources, fetchCloudAccounts bool) (*amsv1.QuotaCostListResponse, error) {
```

Comment:
Can we have the return type as [QuotaCostList](https://pkg.go.dev/github.com/openshift-online/ocm-sdk-go@v0.1.282/accountsmgmt/v1#QuotaCostList) rather than the response? Callers of this method would be interested in the list rather than the response as a whole.